nginx/nginx.tmpl

```diff
@@ -102,9 +104,21 @@ http {
 
 {{- range $entry := .Servers }}
     # ingress: {{ $entry.Name }}
+  {{- range $IngressPortConf := $IngressPorts }}
     server {
-        listen {{ $port }}{{ if $proxyprotocol }} proxy_protocol{{ end }};
+        listen {{ $IngressPortConf.Port }}{{ if $proxyprotocol }} proxy_protocol{{ end }}{{- if eq $IngressPortConf.Name "https" }} ssl {{ end }};
```

Comment:
Does proxy_protocol work with the ssl option?

Reply:
It sould https://www.nginx.com/resources/admin-guide/proxy-protocol/